editoast/Cargo.toml

```diff
@@ -55,7 +58,7 @@ geos.workspace = true
 heck = "0.5.0"
 image = "0.25.1"
 inventory = "0.3"
-iso8601 = "0.6.1"
+iso8601.workspace = true
```

Comment:
Did you check that all the dependencies were still needed in `editoast`? I believe this `iso8601` can be removed from `editoast`.

Reply:
Done.